### PR TITLE
[MC-21] Create Buttom Component

### LINKED ISSUE #21: [Feature] - Create Buttom Component
**Description**
I want a button to be created that will be used as the default for the entire site, following the prototype design.
There are four types of buttons that will be used on the system:

- Primary button
- Outline button
- Add button with icon '+'
- Button with icon at the beginning

**Additional context**
image
image
image

**Acceptance Criteria**
- Separate CSS files
- The code must be responsive
- Build successfully
- lint successfully
- Use color in variables
- Implement all types of buttons

### PR DESCRIPTION
**Pull request by wildphyru**
# Feature/Issue Description
This PR adds Buttons components

# Changes Summary
- Added new buttons.scss file;
- - Added an import to components/style.scss;
- New divs added at index.html;
- Added 2 new image assets:
        Icon-Jogos.png;
        Icon-Usuarios.png;
- Added new font variables at _variables.scss;
- Added 3 new font assets:
        Biryani-Regular.ttf;
        IMFellEnflish-Regular.ttf;
        NotoSans-Regular.ttf;

# How Has This Been Tested?
This application was hosted locally then it was verified by user imput going from Home Page to Components.

# Images (Opcional)
Buttons

### CODE REVIEW

**Review by tvps20 — DISMISSED** [review]

File: src/assets/scss/buttons.scss [hunk 1]
```diff
@@ -0,0 +1,88 @@
+@import './style.scss';
+
+//Border, font, align and shadow >> all buttons
+.button {
+    align-items: center;
+    padding: 0;
+    border: none;
+    outline: none;
+    overflow: hidden;
+    box-shadow: 0px 2px 3px;
```
Comment (pedroemanuel42) [inline src/assets/scss/buttons.scss]: You can add transparency here to let the shadow more smooth

File: src/assets/scss/buttons.scss [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        width: 200px;
+        height: 50px;
+        border-radius: 10px;
+        max-width: 200px !important;
+        min-width: 75px !important;
+    }
+
+    //Accent buttons color
+    &.accent {
+        background: $accent-light;
```
Comment (pedroemanuel42) [inline src/assets/scss/buttons.scss]: background is a shorthand, if you're just setting one attribute is better to use background-color

File: src/assets/scss/buttons.scss [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+        //Hover color for accent buttons
+        &:hover {
+            background: $accent-dark;
```
Comment (pedroemanuel42) [inline src/assets/scss/buttons.scss]: Same shorthand here

File: src/assets/scss/buttons.scss [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        }
+    }
+
+    //Primary buttons color
+    &.primary {
+        background: $primary-light;
```
Comment (pedroemanuel42) [inline src/assets/scss/buttons.scss]: Here as well

File: src/assets/scss/buttons.scss [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+        //Hover color for primary buttons
+        &:hover {
+            background: $primary-dark;
```
Comment (pedroemanuel42) [inline src/assets/scss/buttons.scss]: here.

**Review by pedroemanuel42 — DISMISSED** [review]

File: src/pages/components/main.js [hunk 6]
```diff
@@ -1,5 +1,6 @@
 import { LayoutProviderService } from '../../services/layout-provider.service';
 
+import '../../assets/scss/buttons.scss';
```
Comment (feelsantiago) [inline src/pages/components/main.js]: This import should be add to components/style.scss

**Review by feelsantiago — CHANGES_REQUESTED** [review]

File: src/pages/components/main.js (hunk 6 above)
Comment (wildphyru) [reply src/pages/components/main.js]: It was already there o.o

File: src/assets/scss/buttons.scss (hunk 5 above)
Comment (wildphyru) [reply src/assets/scss/buttons.scss]: Done

File: src/assets/scss/buttons.scss (hunk 4 above)
Comment (wildphyru) [reply src/assets/scss/buttons.scss]: Done

File: src/assets/scss/buttons.scss (hunk 3 above)
Comment (wildphyru) [reply src/assets/scss/buttons.scss]: Done

File: src/assets/scss/buttons.scss (hunk 2 above)
Comment (wildphyru) [reply src/assets/scss/buttons.scss]: Done

File: src/pages/components/main.js (hunk 6 above)
Comment (feelsantiago) [reply src/pages/components/main.js]: The import is on the js file but need to be placed on the scss file.

Comment (wildphyru) [reply src/pages/components/main.js]: Done

File: src/assets/scss/buttons.scss [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+
+//Button text align
+.button-text {
+    justify-content: center;
+    width: 100%;
+}
+
+//Button icon size and padding
+.button-icon {
+    padding: 0 20px;
```
Comment (EvertonDuarteGuimaraes) [inline src/assets/scss/buttons.scss:80]: Eu não tenho certeza, mas acho que tem uma unidade de medida melhor que px pra ficar melhor na responsividade o padding.

**Review by EvertonDuarteGuimaraes — DISMISSED** [review]

**Review by EvertonDuarteGuimaraes — DISMISSED** [review]

**Review by EvertonDuarteGuimaraes — DISMISSED** [review]

File: src/assets/scss/buttons.scss (hunk 7 above)
Comment (wildphyru) [reply src/assets/scss/buttons.scss:80]: Wym?

**Review by pedroemanuel42 — DISMISSED** [review]

File: src/pages/components/style.scss [hunk 8]
```diff
@@ -1,4 +1,5 @@
 @import '@/assets/scss/style.scss';
+@import '../../assets/scss/buttons.scss';
```
Comment (feelsantiago) [inline src/pages/components/style.scss]: Please use import with @ to be consistent

**Review by feelsantiago — CHANGES_REQUESTED** [review]

File: src/pages/components/style.scss (hunk 8 above)
Comment (wildphyru) [reply src/pages/components/style.scss]: This import already have @, I don't know what you mean

Comment (wildphyru) [reply src/pages/components/style.scss]: Done

**Review by feelsantiago — APPROVED** [review]

**Review by EvertonDuarteGuimaraes — APPROVED** [review]

**Pull request merged**